missions/ME_Austria_Missions.txt

```diff
@@ -1,1674 +1,6981 @@
+		has_country_flag = me_hab_german_path
+	}
+	has_country_shield = yes
+
+	me_hab_rhineland = {
+		icon = rhineland_conquest
+		required_missions = {
+			me_hab_swiss
+			me_hab_archduchy
+		}
+		position = 2
+		trigger = {
+			upper_rhineland_area = {
+				type = all
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+			lower_rhineland_area = {
+				type = all
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+			north_rhine_area = {
+				type = all
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+		}
+		effect = {
+			85 = {
+				add_center_of_trade_level = 1
+				add_base_production = 3
+			}
+			ME_give_claims = { region = south_german_region }
+		}
+	}
+}
+
+me_hab_4_1 = {
+	slot = 4
+	generic = no
+	ai = yes
+	potential = {
+		tag = HAB
+		has_country_flag = use_missions_expanded_missions_instead_of_paradox
+	}
+	has_country_shield = yes
+
+	me_hab_carpathia = {
+		icon = king_of_hungary
+		required_missions = {
+			me_hab_budapest
+		}
+		position = 4
+		provinces_to_highlight = {
+			OR = {
+				area = transylvania_area
+				area = southern_transylvania_area
+			}
+			NOT = {
+				country_or_non_sovereign_subject_holds = ROOT
+			}
+		}
+		trigger = {
+			transylvania_area = {
+				type = all
+				country_or_non_sovereign_subject_holds = ROOT
+				NOT = {
+					local_autonomy = 20
+				}
+			}
+			southern_transylvania_area = {
+				type = all
+				country_or_non_sovereign_subject_holds = ROOT
+				NOT = {
+					local_autonomy = 20
+				}
+			}
+		}
+		effect = {
+			custom_tooltip = me_gas_prevent_transylvanian_revolt
+			
+			if = {
+				limit = {
+					transylvania_area = {
+						has_province_modifier = autonomy_decreased
+					}
+				}
+				transylvania_area = {
+					remove_province_modifier = autonomy_decreased
+				}
+			}
+			else = {
+				transylvania_area = {
+					add_province_modifier = {
+						name = me_placated_transylvanian_nobility
+						duration = 7300
+					}
+				}
+			}
+			if = {
+				limit = {
+					southern_transylvania_area = {
+						has_province_modifier = autonomy_decreased
+					}
+				}
+				southern_transylvania_area = {
+					remove_province_modifier = autonomy_decreased
+				}
+			}
+			else = {
+				southern_transylvania_area = {
+					add_province_modifier = {
+						name = me_placated_transylvanian_nobility
+						duration = 7300
+					}
+				}
+			}
+			if = {
+				limit = {
+					4128 = {
+						culture_group = carpathian
+						province_is_or_accepts_culture = yes
+						has_great_project = {
+							type = bran_castle 
+							tier = 0
+						}
+					}
+				}
+				add_great_project_tier = {
+					type = bran_castle 
+					tier = 1
+				}
+			}
+		}
+	}
+	
+	
+	me_hab_crusade = {
+		icon = mission_crusade_for_varna
+		required_missions = {
+			me_hab_carpathia
+		}
+		position = 5
+		provinces_to_highlight = {
+			OR = {
+				area = bulgaria_area
+				area = silistria_area
+			}
+			NOT = { country_or_non_sovereign_subject_holds = ROOT }
+			NOT = { exists = TUR }
+		}
```

Comment:
only highlight those provinces if TUR doesn't exist? 🤨

Reply:
I copied this from you, actually :P

Reply:
what code 😳 I must fix that